uui-core/src/types/props.ts

```diff
@@ -97,6 +97,7 @@ export interface IDropdownToggler extends IHasCaption, IClickable {
     isDropdown?: boolean;
     /** Called when associated dropdown should open or close  */
     toggleDropdownOpening?: (value: boolean) => void;
+    handleClickOutside?: (e: React.FocusEvent<HTMLElement>) => void;
```

Comment:
This handler will be specific for PickerInput case, in any other case dropdown toggler won't receive it, so let's move it to the `PickerTogglerProps` interface.